compiler/rustc_resolve/src/lib.rs

```diff
@@ -749,6 +750,7 @@ impl AmbiguityKind {
             AmbiguityKind::MoreExpandedVsOuter => {
                 "a conflict between a macro-expanded name and a less macro-expanded name from outer scope during import or macro resolution"
             }
+            AmbiguityKind::ExternalCrate => "it has some error in extern crate",
```

Comment:
```suggestion
            AmbiguityKind::External => "multiple glob imports of a name in the same module",
```
Let's reuse the wording from `GlobVsGlob` for now.